pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -272,3 +280,38 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 }
 
 func (alloc *Action) UnInitialize() {}
+
+func setPodVolumesByNode(pod *v1.Pod, node *v1.Node) *volumescheduling.PodVolumes {
+	client := fake.NewSimpleClientset()
+	informerFactory := informers.NewSharedInformerFactory(client, 0)
+	opts := []runtime.Option{
+		runtime.WithClientSet(client),
+		runtime.WithInformerFactory(informerFactory),
+	}
+	fh, err := runtime.NewFramework(nil, nil, nil, opts...)
+	if err != nil {
+		klog.V(3).Infof("new framework handle instance failed for %v", err.Error())
+		return nil
+	}
+	pl, err := volumebinding.New(&config.VolumeBindingArgs{
+		BindTimeoutSeconds: 300,
+	}, fh)
+	if err != nil {
+		klog.V(3).Infof("new volumebinding instance failed for %v", err.Error())
+		return nil
+	}
+
+	p := pl.(*volumebinding.VolumeBinding)
+	var podVolumes *volumescheduling.PodVolumes
+	boundClaims, claimsToBind, _, err := p.Binder.GetPodVolumes(pod)
+	if err != nil {
```

Comment:
check info log is enough to handle the err, should we need return the err?

Reply:
if err is not nil, the podVolumes is nil, It can be the flag to judge if some operation fails.

Reply:
done